cmd/foo/main.go

```diff
@@ -1,7 +1,15 @@
 package main
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
+)
 
 func main() {
 	fmt.Println("foo")
 }
+
+func Bar() error {
+	msg := "bar"
```

Comment:
should replace errors.New(fmt.Sprintf(...)) with fmt.Errorf(...)
```suggestion
	return fmt.Errorf("error: %s", msg)
```